### PR TITLE
APEXMALHAR-2043 using property logViolationsToConsole instead of cons…

### PR DESCRIPTION
**Pull request by otterc**
…oleOutput

### CODE REVIEW

**Comment by otterc** [comment]
@vrozov can you please review this.

File: library/pom.xml [hunk 1]
```diff
@@ -35,6 +35,7 @@
   <name>Apache Apex Malhar (incubating) Library</name>
 
   <properties>
+    <checkstyle.console>false</checkstyle.console>
```
Comment (tweise) [inline library/pom.xml:38]: Why this property if the setting is local and final for the module?

Comment (otterc) [inline library/pom.xml:38]: This is how it is in Apex core as well by @vrozov so was mimicking it.
I think the reason is that by default we don't want to print all the existing violations to console so we set the user property `checkstyle.console` to `false`. However we give the user the option to see the existing violations by `mvn checkstyle:check -Dcheckstyle.console`  if they want to see the existing violations.

If `logViolationsToConsole` is set to false, then user will not be able to spit out violations to the console.

**Comment by tweise** [comment]
The user property does not override the setting in pom.xml?

https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html

**Comment by otterc** [comment]
It doesn't seem to override.  This is how I tried and it doesn't work.

```
<logViolationsToConsole>false</logViolationsToConsole>
```

I ran 
`mvn checkstyle:check -Dcheckstyle.console=true` 
it doesn't print the violations.

**Pull request merged**